charts/flink-job/templates/operator-rolebinding.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
 subjects:
   {{- range $group := .Values.operatorGroups }}
   - kind: Group
-     name: {{ $group }}
-     namespace: {{ .Release.Namespace }}
+    name: {{ $group }}
+    namespace: {{ $.Release.Namespace }}
```

Comment:
we need to split the helm lint and helm test stuff apart...helm test has never worked for this chart and it's masking failures that I think would be caught by helm lint